data-prepper-plugins/opensearch/src/main/resources/index-template/otel-v1-apm-span-index-standard-template.json

```diff
@@ -3,39 +3,136 @@
   "template": {
     "mappings": {
       "date_detection": false,
+      "_source": {
+        "enabled": true
+      },
       "dynamic_templates": [
         {
-          "resource_attributes_map": {
+          "integer_resource_attributes": {
             "mapping": {
-              "type": "keyword"
+              "type": "long"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_resource_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "resource.attributes.*",
+            "match_mapping_type": "double"
+          }
+        },
+        {
+          "default_resource_attributes": {
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 256
             },
             "path_match": "resource.attributes.*"
           }
         },
         {
-          "span_attributes_map": {
+          "integer_scope_attributes": {
             "mapping": {
-              "type": "keyword"
+              "type": "long"
+            },
+            "path_match": "instrumentationScope.attributes.*",
+            "match_mapping_type": "long"
+          }
+        },
+        {
+          "double_scope_attributes": {
+            "mapping": {
+              "type": "double"
             },
-            "path_match": "span.attributes.*"
+            "path_match": "instrumentationScope.attributes.*",
+            "match_mapping_type": "double"
+          }
+        },
+        {
+          "default_scope_attributes_map": {
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 256
+            },
+            "path_match": "instrumentationScope.attributes.*"
+          }
+        },
+        {
+          "long_attributes": {
+            "mapping": {
+              "type": "long"
+            },
+            "path_match": "attributes.*"
+          }
+        },
+        {
+          "double_attributes": {
+            "mapping": {
+              "type": "double"
+            },
+            "path_match": "attributes.*"
+          }
+        },
+        {
+          "default_attributes": {
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 256
+            },
+            "path_match": "attributes.*"
           }
         }
       ],
-      "_source": {
-        "enabled": true
-      },
       "properties": {
+        "droppedEventsCount": {
```

Comment:
`dropperAttributesCount`. Also can you validate the existing mapping with 
https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto

I see a lot of missing pieces.

Reply:
we did some analysis here: https://github.com/opensearch-project/opensearch-catalog/issues/218

Reply:
I am going to do that.